types/rdfjs__parser-jsonld/rdfjs__parser-jsonld-tests.ts

```diff
@@ -14,6 +13,6 @@ const parser3 = new Parser({ baseIRI, context, factory });
 
 const sink: Sink = parser1;
 
-const eventEmitter1: EventEmitter = parser1.import(stream);
-const eventEmitter2: EventEmitter = parser1.import(stream, {});
-const eventEmitter3: EventEmitter = parser1.import(stream, { baseIRI, context, factory });
+const eventEmitter1: Stream = parser1.import(stream);
+const eventEmitter2: Stream = parser1.import(stream, {});
+const eventEmitter3: Stream = parser1.import(stream, { baseIRI, context, factory });
```

Comment:
Should also test with a different quad implementation.

Reply:
Something like this?
Have to go all the way back to a custom quad. Using `BaseQuad` is cheating imo ;)

```ts
interface SpecializedQuad extends BaseQuad {}
const typedStream: Stream<SpecializedQuad> = <any> {};
const typedParser: Parser<SpecializedQuad> = <any> {};
const typedImported: Stream<SpecializedQuad> = typedParser.import(typedStream);
const typedImported1: Stream<SpecializedQuad> = typedParser.import(typedStream, {});
const typedImported2: Stream<SpecializedQuad> = typedParser.import(typedStream, { baseIRI, context, factory });
```

Reply:
Yep!